app/models/admin.rb

```diff
@@ -6,22 +6,23 @@ class Admin < ApplicationRecord
     :owner,
     :supervisor,
     :analyst,
-    :organization_owner
+    :organization_owner,
+    :healthcare_counsellor
```

Comment:
We are going to totally redo permissions later, but let's reduce this to just `counselor` for now.

Reply:
ok